blog/posts/making-the-remix-austin-blog.mdx

```diff
@@ -0,0 +1,214 @@
+import mattConfused from "./img/matt-confused.jpg";
+import refreshFunctionality from "./img/refresh.jpg";
+import mattAuthor from "./img/matt-author.png";
+
+<BlogImage src={blogBanner} alt="Blog banner" />
+
+## So how did we get here?
+
+Well, it starts a little before I got here. If I'm going to play historian, I'll say it all
+began with a [discussion](https://github.com/remix-austin/remixaustin-com/discussions/1).
+As a subset of the greater Remix community, there was talk of what exactly this site was going
+to be other than a place to highlight the date of the next Austin Remix Meetup. What kind of
+greater impact did we want to have on all the Remixers out there?
+
+The rumblings of having a blog began back there, but they really started to move from idea
+to conception when our good friend Colby White [decided to chime in with a plan](https://github.com/remix-austin/remixaustin-com/discussions/89).
+A few things began to take shape at that point, some more discussion, an [early PR with mock
+blog content](https://github.com/remix-austin/remixaustin-com/pull/86). But from what I can tell,
+nothing solidified just yet.
+
+## Where did I come in?
+
+A month after Colby White's discussion of what a blog implementation of Remix could be, that's when I
+showed up. As a developer trying to get back into the React community after spending 3 years doing
+nothing but Angular at my full time job (seriously y'all, Angular is not cool and I want out 😅),
+I needed a way to brush up my old skills.
+
+I went to a few React meetups in the Austin area, and through the word of mouth heard about the
+Remix Austin meetup. I showed up, heard some awesome talks, hung out with cool people, and figured
+this was a good way to get back in it.
+
+## Steal like an artist
+
+I'm a big fan of using a file system as a CMS, and the concept has been around for a while now. I especially
+like file-based CMS-es for proof of concept work on blogs that are just getting started. This site is the
+perfect fit.
+
+I was happy to see in Colby's discussion that he also liked the file system approach to a degree and
+had gone down that route when making his [own website](https://colbywhite.dev/).
+He used `.mdx` files to write markdown posts with a little bit of React flavor.
+
+You might be thinking, "Well problem already solved! Remix has support for `.mdx` files!"
+
+And you're right, but even Remix acknowledges in [their documentation](https://remix.run/docs/en/main/guides/mdx)
+that their implementation is not designed to scale and that it's better to use `mdx-bundler` to build content.
+A few people in the Remix community have already made their own [creative solutions](https://kentcdodds.com/blog/how-i-built-a-modern-website-in-2021#mdx-compilation-with-mdx-bundler)
+to this problem. These solutions work, and Colby's solution, inspired by [this Remix stack](https://github.com/freekrai/remix-blog),
+is hosted on [his Github](https://github.com/colbywhite/colbywhite.dev).
```

Comment:
Just a thought, maybe use a different word than "hosted on". To me that sounds like you're saying he is using GitHub to host his website (like how we're using fly). I assume you just mean "can be found on his GitHub" or something.

Reply:
Yeah, the second point is what I was getting at. Fixing it now.